catalogue/models/component/__init__.py

```diff
@@ -128,6 +124,69 @@ class Meta:
         verbose_name = "Vorraussetzungen"
         verbose_name_plural = verbose_name
 
+    def __str__(self):
+        return ""
+
+
+class Contact(models.Model):
+    # Contact Information
+    contact_manufacturer = models.CharField(
+        "Hersteller",
+        help_text="Entwickler und/oder Hersteller der Lösung",
+        max_length=1000,
+    )
+    contact_person_name = models.CharField(
+        "Name",
+        help_text="Name des zuständigen Mitarbeiters",
+        max_length=100,
+    )
+    contact_address_zip = models.CharField(
+        "PLZ",
+        help_text="PLZ in der sich der Anbieter befindet",
+        max_length=10,
+        blank=True,
+    )
+    contact_address_country = models.CharField(
+        "Land",
+        help_text="Land in dem sich der Anbieter befindet",
+        max_length=100,
+        default="Deutschland",
+    )
+    contact_address_city = models.CharField(
+        "Stadt",
+        help_text="Stadt in der sich der Anbieter befindet",
+        max_length=100,
+        blank=True,
+    )
+    contact_address_street = models.CharField(
+        "Straße",
+        help_text="Straße des Anbieters",
+        max_length=100,
+        blank=True,
+    )
+    contact_email = models.EmailField(
+        "Email",
+        help_text="Email Adresse des zuständigen Mitarbeiters",
+        max_length=100,
+        blank=True,
+    )
```

Comment:
reorder fields as this reflects in admin.
- person
- email of person
- street
- city
- zip
- country